django_event_sourcing/registers.py

```diff
@@ -38,9 +43,9 @@ def decorator(f):
 
         return decorator
 
-    def register_side_effect(self, *, side_effect):
+    def register_side_effect(self, *, side_effect, condition=None):
```

Comment:
```suggestion
    def register_side_effect(self, callable, *, condition=None):
```

I think this is a nicer interface because the callable is the only required argument:

```py
register_side_effect(send_emails, condition=CanSendEmails)
```